manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
@dcarley Sorry, I missed your comment as it's after Colins comment even if it's younger. 
We can go case by case, analyse why we need private keys, check if it's possible to not provide them and if they are necessary make sure they are deleted after usage.